kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/UserTaskServiceIntegrationTest.java

```diff
@@ -1269,7 +1343,7 @@ public void testFindTaskEvents() throws Exception {
 
     @Test
     public void testFindTaskEventsForNotExistingTask() {
-        String invalidId = "-9,999";
+        String invalidId = "-9.999";
```

Comment:
Will ```.``` work as a thousand delimiter?

Reply:
I believe this will be locale specific so better to use lower number so the delimiter won't be needed

Reply:
I changed the thousand delimiter because this assertion failed: KieServerAssert.assertResultContainsString(e.getMessage(),  "Could not find task instance with id \"" + invalidId + "\""); due to the conversion of -9999L here: taskClient.findTaskEvents(CONTAINER_ID, -9999L, 0, 10);
I've suggested what Maciej said using a lower number.